roles/easy-build-packages/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+#
+# install software modules using EasyBuild:
+#
+---
+- name: "install depending software"
+  become: true
+  apt:
```

Comment:
Should we have a check in here to throw an error on non-Ubuntu systems?

Reply:
@issamsaid : Do we need to add the RHEL equivalent here?

Reply:
We should split this to do the right pre-install tasks for each distro type, similar to the pattern used here: https://github.com/NVIDIA/deepops/tree/master/roles/nvidia-ml/tasks